### PR TITLE
Add Dead Man's Refuge

### PR DESCRIPTION
**Pull request by Ravernstal**

### CODE REVIEW

**Comment by Ravernstal** [comment]
Hi, since this mod is also published on Bethesda.net for PC, Xbox, and PS4, should I add three extra entries to the URL section?

**Comment by MacSplody** [comment]
> Hi, since this mod is also published on Bethesda.net for PC, Xbox, and PS4, should I add three extra entries to the URL section?

Adding the link for the PC version would be fine, not much use for the Xbox or PS4 versions.

**Comment by MacSplody** [comment]
You'll have to move the new metadata, it's currently in the middle of another mods files.

image

It should look like below.

```yaml
###### Locations - New Dungeons ######
  - name: '(CauseReducedCut(- Ayleid Gear Consistency Patch)?|Rielle - Vault of Meridia)\.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/66676/'
        name: 'The Cause - Reduced Cut'
  - name: 'CauseReducedCut.esp'
    dirty:
      - <<: *quickClean
        crc: 0x6634DF7A
        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 11
  - name: 'CauseReducedCut- Ayleid Gear Consistency Patch.esp'
    clean:
      - crc: 0x51DD187F
        util: 'SSEEdit v4.0.4b'
  - name: 'Rielle - Vault of Meridia.esp'
    clean:
      - crc: 0x114081E6
        util: 'SSEEdit v4.0.4b'

  - name: 'DeadMansRefuge.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/66194'
        name: 'Dead Man''s Refuge'
    msg:
      - <<: *patchIncluded
        subs: [ '[Arena - An Encounter Zone Overhaul](https://www.nexusmods.com/skyrimspecialedition/mods/33487)' ]
        condition: 'active("Arena.esp") and not active("DeadMansRefugeArenaPatch.esp")'
    clean:
      - crc: 0x683B8666
        util: 'SSEEdit v4.0.4'
  - name: 'DeadMansRefugeArenaPatch.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/66194'
        name: 'Dead Man''s Refuge - Arena Patch'
    clean:
      - crc: 0x43F3A723
        util: 'SSEEdit v4.0.4'

###### Locations - New Lands ######
```

**Comment by Ravernstal** [comment]
My bad. It should be fixed now and I've added the Bethesda.net link.

File: masterlist.yaml [hunk 1]
```diff
@@ -17332,6 +17332,29 @@ plugins:
...
+        util: 'SSEEdit v4.0.4'
+  - name: 'DeadMansRefugeArenaPatch.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/66194'
+        name: 'Dead Man''s Refuge - Arena Patch (Nexus Mods)'
+      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/4283327'
+        name: 'Dead Man''s Refuge - Arena Patch (Bethesda.net)'
+    clean:
+      - crc: 0x43F3A723
+        util: 'SSEEdit v4.0.4'
```
Comment (MacSplody) [inline masterlist.yaml]: Okay, just needs a few format changes for the locations.

```suggestion
  - name: 'DeadMansRefuge(ArenaPatch)?\.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/66194/'
        name: 'Dead Man''s Refuge (Nexus Mods)'
  - name: 'DeadMansRefuge.esp'
    url:
      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/4283320/'
        name: 'Dead Man''s Refuge (Bethesda.net)'
    msg:
      - <<: *patchIncluded
        subs: [ '[Arena - An Encounter Zone Overhaul](https://www.nexusmods.com/skyrimspecialedition/mods/33487)' ]
        condition: 'active("Arena.esp") and not active("DeadMansRefugeArenaPatch.esp")'
    clean:
      - crc: 0x683B8666
        util: 'SSEEdit v4.0.4'
  - name: 'DeadMansRefugeArenaPatch.esp'
    url:
      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/4283327/'
        name: 'Dead Man''s Refuge - Arena Patch (Bethesda.net)'
    clean:
      - crc: 0x43F3A723
        util: 'SSEEdit v4.0.4'
```

Comment (MacSplody) [reply masterlist.yaml]: You can use the commit suggestion button to apply changes.

image

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
Thanks.

**Pull request merged**